include/hal/library/cryptlib/cryptlib_ec.h

```diff
@@ -155,6 +155,16 @@ extern bool libspdm_ecdsa_sign(void *ec_context, size_t hash_nid,
                                const uint8_t *message_hash, size_t hash_size,
                                uint8_t *signature, size_t *sig_size);
 
+#if LIBSPDM_FIPS_MODE
+/**
+ * Carries out the EC-DSA signature for fips: hardcode random number.
+ *
+ **/
+extern bool libspdm_ecdsa_sign_fips(void *ec_context, size_t hash_nid,
+                                    const uint8_t *message_hash, size_t hash_size,
+                                    uint8_t *signature, size_t *sig_size);
+#endif/*LIBSPDM_FIPS_MODE*/
```

Comment:
This does not make sense.

It says: hardcode random number.
But it does not describe how to let the caller input random number value.

Reply:
Thanks. I have added a new input parameter(function point) to hardcode the random number.
And both OpenSSL and MbedTLS  tests are passed.

Reply:
I suggest `libspdm_ecdsa_sign_fips` -> `libspdm_ecdsa_sign_ex`.
`hardcode_random` -> `random_func`.

Reply:
Thanks. I have updated the function names and comments.